src/ptvsd/adapter/messages.py

```diff
@@ -247,6 +247,34 @@ def disconnect(self):
                 # for us to respond, there's no rush.
                 debuggee.terminate(after=60)
 
+    @_only_allowed_while("running")
+    def pause_request(self, request):
+        request.arguments['threadId'] = '*'
+        self._server.delegate(request)
+        return {}
+
+    @_only_allowed_while("running")
+    def continue_request(self, request):
+        request.arguments['threadId'] = '*'
+        self._server.delegate(request)
+        return {'allThreadsContinued': True}
+
+    def on_ptvsd_systemInfo(self, request):
+        sys_info = {
+            'ptvsd': {
+                'version': ptvsd.__version__,
+            },
+        }
+
+        try:
+            result = self._server.send_request('pydevdSystemInfo').wait_for_response()
+            sys_info.update(result)
+        except messaging.MessageHandlingError as exc:
+            raise messaging.MessageHandlingError(
```

Comment:
A shorthand for this is:
```py
request.cant_handle("..." + str(exc))
```
Aside from being shorter, this also has the advantage that the raised exception will have `cause` set to `request` (not that it matters in this case, either way it'll work).